.tekton/pull-request.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
-  generateName: on-pr-
+  name: has-on-pull-request
   namespace: application-service
```

Comment:
you don't need to specify the namespace either it will be tested where the repository is